### ISSUE TITLE
Why did containerd delete all my permissions from ttrpc-rust

### ISSUE THREAD

**Issue by Tim-Zhang** [containerd/ttrpc-rust#187]
As a core maintainer of ttrpc-rust I found that my permissions in ttrpc-rust has gone. 
I don't know why this could happen without even a single notify, I want to know who did this and why.

The only thing I know is @fuweid told me that containerd will take action to change permission management to non-core project hours ago through Wechat.

**Comment by fuweid** [comment]
I think it is related to security concern. Thanks for your contribution to ttrpc-rust project. 
I suggest that the admin permissions should be assigned to maintainers who is in the  https://github.com/containerd/ttrpc-rust/blob/master/MAINTAINERS, which is trustable. I will file a pr to promote you as maintainer and add @wllenyj as reviewer.

**Comment by dmcgowan** [comment]
@Tim-Zhang sorry about that, that was me. I bumped any outside collaborator with admin access to maintain access as part of a permission audit we were doing. I realize it may not have been communicated clearly about timing, but looks like we are getting it straightened out in the `MAINTAINERS` file now. I have created the teams in our Github org to reflect that. The @containerd/ttrpc-rust-committers team for maintain permissions and @containerd/ttrpc-rust-reviewers for triage.
